File: Project_Api/Entities/ObjetiveAccount.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Project_Api.Entities
{
    public class ObjetiveAccount
    {
        [Key]
        public int Id { get; set; }
        public int SavingsAccountId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public decimal Fee { get; set; }
        public string Objective { get; set; }
        public decimal Balance { get; set; }
        public float AcumInterest { get; set; }
        public string DaysOfDeposit { get; set; }
        public bool Active { get; set; }

    }
}

File: FrontEnd/src/models/ObjetiveAccount.ts
export default class ObjetiveAccount{
    id: number|undefined;
    SavingsAccountId: number;
    startDate: Date;
    endDate: Date;
    fee: number;
    objetive: string;   
    balance: number;
    acumInterest: number;
    daysOfDeposit: number;
    active: boolean;
    user: number;

    constructor(id: number|undefined, SavingsAccountId: number, startDate: Date, endDate: Date, fee: number, objetive: string, balance: number, acumInterest: number, daysOfDeposit: number, active: boolean, user: number ){
        this.id = id;
        this.SavingsAccountId = SavingsAccountId;
        this.startDate = startDate;
        this.endDate = endDate;
        this.fee = fee;
        this.objetive = objetive;
        this.balance = balance;
        this.acumInterest = acumInterest;
        this.daysOfDeposit = daysOfDeposit;
        this.active = active;
        this.user = user;
    }
}